src/style.css

```
@@ -0,0 +1,104 @@
+:root {
+  --greyColor: #3d4e61;
+  --lightColor: #e1e7ed;
```

IMHO, this is better (because we see the `--c' prefix for and know that it is a variable with a color)


```suggestion
  --c-gray: #3d4e61;
  --c-light: #e1e7ed;
```